release/jobs/health_monitor/templates/health_monitor_ctl

```diff
@@ -27,6 +28,10 @@ case $1 in
       cp $CONFIG_DIR/uaa.pem $STORE_DIR
     fi
 
+    if [ "$HM_SYSLOG_EVENT_FORWARDER_ENABLED" == "true" ]; then
```

Comment:
We did this so we wouldn't need to make the `ctl` script an `.erb` which needs to be rendered. That was propagated some time ago by Dr Nic and his bosh-gen. If you're fine with making the `ctl` a template we can do that.